src/svcutilcore/pkg/svcutilcore.pkgproj

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <ServiceModelFacade>$(BaseIntermediateOutputPath)shims/netcoreapp/facades/System.ServiceModel.dll</ServiceModelFacade>
+    <!-- Skip package validation since this package will now contain the shims and this is not allowed by validation -->
+    <SkipValidatePackage>true</SkipValidatePackage>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\svcutilcore.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
+    </ProjectReference>
+  </ItemGroup>
+  <PropertyGroup>
```

Comment:
Comparing with [Microsoft.XmlSerializer.Generator.pkgproj](https://github.com/dotnet/corefx/blob/master/src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj#L8), I see we don't have ReleaseNotes and ProjectUrl nodes. Do you plan to add them in a new PR? If so, please open an issue for it so we don't forget.

Reply:
#2587